gr-uhd/python/uhd/bindings/usrp_sink_python.cc

```diff
@@ -57,10 +57,13 @@ void bind_usrp_sink(py::module& m)
              D(usrp_sink, set_start_time))
 
 
-        .def("get_usrp_info",
-             &usrp_sink::get_usrp_info,
-             py::arg("chan") = 0,
-             D(usrp_sink, get_usrp_info))
+        .def(
+            "get_usrp_info",
+            [](usrp_sink& self, const size_t chan = 0) {
+                return static_cast<std::map<std::string, std::string>>(
+                    self.get_usrp_info(chan));
+            },
+            py::arg("chan") = 0)
```

Comment:
Except for `usrp_sink` this is exactly the same code as in `usrp_source`.
Can you move this to `usrp_block`?

Reply:
Good point - no reason to have duplicate code.  Second commit is added to address this.